pages/inspiration.js

```diff
@@ -65,16 +67,21 @@ export default function Inspiration() {
     } else {
       return (
         <>
-          <ModalComp btnText='Create Inspiration' title='Create Inspiration'>
-            <CreateInspo user={user} />
-          </ModalComp>
-          {data.map((d) => (
-            <div key={d.id} className='text-white'>
-              <h1>{d.name}</h1>
-              <h3>{d.industry}</h3>
-              <p>{d.content}</p>
-            </div>
-          ))}
+        <GlobalHeader />
```

Comment:
Try putting `GlobalHeader` and Footer in `_app.js` so that you do not have to put it on every page. 

The Global stuff as well as the div wrapper should take care of the whole app.

Reply:
Worked out together!